modules/storage-persistent-storage-nfs-provisioning.adoc

```diff
@@ -70,19 +70,25 @@ metadata:
   name: nfs-claim1
 spec:
   accessModes:
-    - ReadWriteOnce <1>
+    - ReadWriteOnce
   resources:
     requests:
-      storage: 5Gi <2>
+      storage: 5Gi <1>
+  volumeName: pv0001
+  storageClassName: ""
 ----
-<1> As mentioned above for PVs, the `accessModes` do not enforce security, but
-rather act as labels to match a PV to a PVC.
```

Comment:
I probably wasn’t clear enough in my note about removing the first clause in the callout note. I still think the information about access modes is good to have in there, but we just don’t need to say “As mentioned above for PVs”. Would you mind putting the <1> callout and the callout note back in like this?

```
spec:
  accessModes:
    - ReadWriteOnce <1>
  resources:
    requests:
      storage: 5Gi <2>
  volumeName: pv0001
  storageClassName: ""
----
<1> The access modes do not enforce security, but rather act as labels to match a PV to a PVC.
```